ramls/invoice_audit_event.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Invoice audit event",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID of the event",
+      "$ref": "common/uuid.json"
+    },
+    "action": {
+      "description": "Action for invoice (Create, Edit or Delete)",
+      "type": "string",
+      "$ref": "event_action.json"
+    },
+    "invoiceId": {
+      "description": "UUID of the invoice",
+      "$ref": "common/uuid.json"
+    },
+    "userId": {
+      "description": "UUID of the user who performed the action",
+      "$ref": "common/uuid.json"
+    },
+    "eventDate": {
+      "description": "Date time when event triggered",
+      "format": "date-time",
+      "type": "string"
+    },
+    "actionDate": {
+      "description": "Date time when invoice action occurred",
+      "format": "date-time",
+      "type": "string"
+    },
+    "invoiceSnapshot": {
+      "description": "Full snapshot of the invoice",
+      "type": "object",
+      "javaType": "java.lang.Object"
+    }
+  },
+  "additionalProperties": false
```

Comment:
if this even structure changes in the futures, this dto also have to changes because of "additionalProperties": false. 
this field don't allow extra property in event. default true (if additionalProperties is not exists)

Reply:
> if this even structure changes in the futures, this dto also have to changes because of "additionalProperties": false. this field don't allow extra property in event. default true (if additionalProperties is not exists)

So you recommend to set it to `true`?

Reply:
yes